cli/checkpoint/checkpoint.go

```diff
@@ -23,36 +24,23 @@ type Opts struct {
 // Cat print the contents of .snap/.xlog files.
 // Returns an error if such occur during reading files.
 func Cat(tntCli cmdcontext.TarantoolCli) error {
-	var errbuff bytes.Buffer
 	cmd := exec.Command(tntCli.Executable, "-")
-	cmd.Stderr = &errbuff
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
 
-	stdoutPipe, err := cmd.StdoutPipe()
-	if err != nil {
-		return err
-	}
 	stdinPipe, err := cmd.StdinPipe()
 	if err != nil {
 		return err
 	}
 	stdinPipe.Write([]byte(catFile))
 	stdinPipe.Close()
 
-	if err := cmd.Start(); err != nil {
-		return fmt.Errorf("result of cat: %s", errbuff.String())
+	if err := cmd.Run(); err != nil {
```

Comment:
`Run` waits for the command to complete. There is no need to keep the `Wait` call below.